doc/examples/example3.md

```diff
@@ -0,0 +1,18 @@
+Hash {#example3}
+====================
+Hash
+====
+
+The following example generates a SHA1 type.
+To generate a different hash type the procedure is the same. 
```

Comment:
"To generate a different hash type the procedure is the same, we just need to call the specific method (sha256(), sha512())."